pandas/core/groupby/ops.py

```diff
@@ -461,7 +462,15 @@ def wrapper(*args, **kwargs):
 
         return func
 
-    def _cython_operation(self, kind, values, how, axis, min_count=-1, **kwargs):
+    def _cython_operation(
+        self, kind, values, how, axis, min_count=-1, **kwargs
```

Comment:
kind and how are both str

Reply:
Sounds good. I didn't annotate the others because they didn't work as expected; going to take a while to unravel that so leaving to a separate exercise